packages/css/recipes/chip-tabs.css

```
@@ -55,54 +55,54 @@
     gap: 0px;
 }
 .seed-chip-tabs__trigger--variant_neutralSolid {
-    background-color: transparent;
-    color: var(--seed-color-fg-neutral);
+    background-color: var(--seed-box-background, transparent);
+    color: var(--seed-box-color, var(--seed-color-fg-neutral));
     font-weight: var(--seed-font-weight-bold);
 }
 .seed-chip-tabs__trigger--variant_neutralSolid:is(:selected, [data-selected]) {
```

Biome (2.1.2)

[error] Unexpected unknown pseudo-class selected 

See MDN web docs for more details.

(lint/correctness/noUnknownPseudoClass)